avocado-core/pom.xml

```diff
@@ -125,5 +125,21 @@
       <artifactId>scalatest_${scala.version.prefix}</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.seqdoop</groupId>
+      <artifactId>hadoop-bam</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.seqdoop</groupId>
+      <artifactId>cofoja</artifactId>
```

Comment:
Nah, this is the same Cofoja. Cofoja is a won't fix for Java 8, IIRC, but was removed from the latest version of HTSJDK/Picard. _Howaaaaver,_ no one has cut a new release of HTSJDK yet. See https://github.com/samtools/htsjdk/pull/115#issuecomment-72462083.